test/cli/package-import-parent-package-conflict/parent/__package.rb

```diff
@@ -0,0 +1,5 @@
+# typed: strict
+
+class Parent < PackageSpec
+  export Parent
```

Comment:
Ah, this is legal? So this would appear in an importing package as `Parent` and not `Parent::Parent`?

Reply:
If `Parent` exported `Parent::Inheritance`, would that not be a problem? Because then it would be defining `Parent::Inheritence` and not the root level `Parent`?

Reply:
* `Parent` exporting `Parent` is legal, and yes it would appear in an importing package as `Parent`.
* If `Parent` exported `Parent::Inheritance`, that's okay, because the importing package would define `Parent::Inheritance` and `Parent::Child` separately, and nothing would get redefined.